README.md

```diff
@@ -28,31 +28,159 @@ flag is highly recommended for those new to [pip](https://pip.pypa.io).
 A PEX executable (Windows not supported) and source releases are
 [here](https://github.com/planetlabs/planet-client-python/releases/latest).
 
-## Documentation
+## Authentication
 
-Online documentation:
-https://planetlabs.github.io/planet-client-python/index.html
+Planet's APIs require an account for use.
+[Sign up here](https://www.planet.com/explorer/?signup).
 
-Documentation is also provided for download
-[here](https://github.com/planetlabs/planet-client-python/releases/latest).
+Authentication information can be obtained and stored locally with the CLI
+`planet init` command. Both the Python library and CLI will default to using
+the stored authentication information, if available (not yet implemented, see
+issues #244 and #248).
 
+Additionally, authentication information can be provided to the CLI by
+storing the api key in the environmental variable `PL_API_KEY` or by specifying
+the api key with the flag `-k` or `--api-key` (TODO: link to cli documentation
+here).
 
-## Development
+## Quick Start
 
-To contribute or develop with this library, see
-[CONTRIBUTING](https://github.com/planetlabs/planet-client-python/CONTRIBUTING.md)
+### Python Library Usage
 
+The client modules within the Python library are asynchronous, which greatly
+speeds up many interactions with Planet's APIs. Support for asynchronous
+development is native to Python 3.6+ via the
+[`asyncio` module](https://docs.python.org/3/library/asyncio.html) and is
+fairly easy to implement.
+
+```python
+import asyncio
+import os
+
+import planet
+
+API_KEY = os.getenv('PL_API_KEY')
+
+image_ids = ['3949357_1454705_2020-12-01_241c']
+order_details = planet.OrderDetails(
+    'test_order',
+    [planet.Product(image_ids, 'analytic', 'psorthotile')]
+)
+
+async def create_order(order_details):
+    async with planet.Session(auth=(API_KEY, '')) as ps:
+        client = planet.OrdersClient(ps)
+        return await client.create_order(order_details)
+
+oid = asyncio.run(create_order(order_details))
+print(oid)
+```
 
-## API Key
+Not into async? No problem. Just wrap the library and async operations together
+and call from your synchronous code.
 
-The API requires an account for use. [Signup here](https://www.planet.com/explorer/?signup).
+```python
+def sync_create_order(order_details):
+    return asyncio.run(create_order(order_details))
 
-This can be provided via the environment variable `PL_API_KEY` or the flag `-k` or `--api-key`.
+oid = sync_create_order(order_details)
+print(oid)
+```
 
-Using `planet init` your account credentials (login/password) can be used to obtain the api key.
+When using `asyncio.run` to develop synchronous code with the async library,
+keep in mind this excerpt from the
+[asyncio.run](https://docs.python.org/3/library/asyncio-task.html#asyncio.run)
+documentation:
+
+"*This function always creates a new event loop and closes it at the end. It
```

Comment:
Consider using a Markdown blockquote here instead of the custom italics and double quotes.

Reply:
This is stylistic and can be resolved later.